File: Desafios de codigo/desafio cotlin/desafio.kt
object ReceitFederal {
    fun CalcularImposto(salario: Double): Double {
        val aliquota = when {
            (salario >= 0 && salario <= 1100) -> 0.05
            (salario >= 1100 && salario <= 2500) -> 0.10
            else -> 0.15

            else -> TODO("Criar as condiçôes para as aliquota de 10.00% e 15.00%)
        }
        return aliquota * salario
    }
}

fun main() {
    val valoeSalario = realine()!!.toDouble();
    val valorBeneficio = realine()!!.toDouble();
    val valorImposto ReceitFederal.calcularImposto(valorSalario);
    val saida = valorSalario - valorImposto + valorBeneficio;

    println(string.format("%.2f", saida));
}

File: Desafios de codigo/desafio javascript/desafio.js
const valorSalario = parseFloat(get());
const valorBeneFicios = parseFloat(get());

const valorImposto = calcularImposto(valorSalario);

const saida = valorSalario- valorImposto + valorBeneFicios;
print(saida.toFixed(2));

function calcularImposto(salario) {
    let aliquota;
    if (salario >= 0 && salario <=1100){
        aliquota= 0.05;
    } else if (salario >= 1100.01 && salario <=2500.00){
        aliquota = 0.10;
    } else {
        aliquota = 0.15;
    }

    //TODO
    return aliquota * salario;
}